src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -235,6 +235,29 @@ public static GenomeLocSortedSet loadIntervals(
         return sortAndMergeIntervals(genomeLocParser, allIntervals, intervalMergingRule);
     }
 
+    public static List<GenomeLoc> loadIntervalsNonMerging(
```

Comment:
Instead of creating this separate method, couldn't you use the existing `sortAndMergeIntervals()` method, and just modify it to special-case NONE as the interval merging rule? Eg., modify `sortAndMergeIntervals()` to not call `mergeIntervalLocations()` if the rule is NONE, and instead just sort? Or is that problematic in some way?

Reply:
(If you do need to keep this method around for some reason, as punishment you must write full javadoc for it)